docs/layers/hexagon-layer.md

```diff
@@ -75,6 +77,15 @@ to number of counts by passing in an arbitrary color domain. This property is ex
 Hexagon color ranges as an array of colors formatted as `[[255, 255, 255, 255]]`. Default is
 [colorbrewer](http://colorbrewer2.org/#type=sequential&scheme=YlOrRd&n=6) `6-class YlOrRd`.
 
+##### `colorValueAccessor` (Function, optional)
```

Comment:
when getColorValue is not defined by layer, the BinSorter used by layer will have getValue = points =>  points.length by default.  I don't know if it's necessary to define it twice